keyboards/chlx/merro60/keymaps/via/merro60.json

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
This file doesn't belong in the QMK repo, and will not be used by QMK.   It needs to be removed, and PR'ed to `the-via`'s keyboards repo.

Reply:
removed file

Reply:
PR'd JSON file to via